package.json

```diff
@@ -87,43 +87,43 @@
     ]
   },
   "dependencies": {
-    "@commitlint/cli": "7.6.1",
-    "@commitlint/config-conventional": "7.6.0",
-    "@oclif/command": "1.5.13",
-    "@oclif/config": "1.13.0",
-    "@oclif/plugin-help": "2.1.6",
-    "@semantic-release/commit-analyzer": "6.1.x",
-    "@semantic-release/git": "7.0.8",
-    "@semantic-release/github": "5.2.10",
-    "@semantic-release/npm": "5.1.7",
-    "@semantic-release/release-notes-generator": "7.1.x",
-    "cli-ux": "5.2.1",
+    "@commitlint/cli": "^8.0.0",
+    "@commitlint/config-conventional": "^8.0.0",
+    "@oclif/command": "^1.5.16",
+    "@oclif/config": "^1.13.2",
+    "@oclif/plugin-help": "^2.2.0",
+    "@semantic-release/commit-analyzer": "^6.2.0",
+    "@semantic-release/git": "^7.0.16",
+    "@semantic-release/github": "^5.4.2",
```

Comment:
I'm wondering whether https://github.com/semantic-release/github/releases?after=v4.0.1 does not affect us.